edm4hep.yaml

```diff
@@ -503,6 +503,27 @@ datatypes:
       void setCovMatrix(float value, edm4hep::Cartesian dimI, edm4hep::Cartesian dimJ) { getCovMatrix().setValue(value, dimI, dimJ); }\n
       "
 
+  edm4hep::SensitiveWireHit:
+    Description: "Sensitive wire hit, before the hit position degeneracy is resolved. The circle representing possible positions is parametrized with its center, radius and normal vector."
+    Author: "EDM4hep authors"
+    Members:
+      - uint64_t cellID // ID of the sensor that created this hit
+      - int32_t type // type of the raw data hit
+      - int32_t quality // quality bit flag of the hit
+      - float time [ns] // time of the hit
+      - float eDep [GeV] // energy deposited on the hit
+      - float eDepError [GeV] // error measured on eDep
+      - edm4hep::Vector3d position [mm] // point on the sensitive wire which is closest to the hit (center of the circle)
+      - edm4hep::Vector3d directionSW // direction of the sensitive wire (normal vector to the circle)
+      - float distanceToWire [mm] // distance hit-wire (radius of the circle)
+    VectorMembers:
+      - uint16_t nElectrons // number of electrons for each cluster (number of clusters = vector size)
```

Comment:
What is the typical number of electrons per cluster? I suppose 65535 is enough to work in all cases?

Reply:
Yes, `uint16` will do the job, this value is typically below 10.

Reply:
> What is the typical number of electrons per cluster? I suppose 65535 is enough to work in all cases?

According to the paper they published [1], Fig 15, right plot:
![image](https://github.com/user-attachments/assets/f47938bd-52d2-4e58-b50a-746897c7a527)
There is a tail that goes up to 450, but the probability is extremely low, most of the clusters have less than 10 electrons as Brieuc said.

[1] https://arxiv.org/pdf/2105.07064

Reply:
450 still fits into 9 bits, so there is a 7 bit safety margin. I think this is OK. Thanks for the reference.